internal/job/vault_seal_watcher.go

```diff
@@ -30,35 +32,140 @@ func (v *VaultSealWatcher) CronSpec() string {
 
 func (v *VaultSealWatcher) Run() {
 	v.log.Debug("started vault seal watcher job")
-	vc, err := client.NewVaultClient(v.log, v.conf)
-	if err != nil {
-		v.log.Errorf("%s", err)
-		return
-	}
 
-	res, err := vc.IsVaultSealed()
-	if err != nil {
-		v.log.Errorf("failed to get vault seal status, %s", err)
-		return
-	}
+	if v.conf.HAEnabled {
+		v.log.Infof(" Vault HA ENABLED", v.conf.HAEnabled)
 
```

Comment:
keep support for single vault, HA vault,

if single vault, then existing solution should work.
if HA then new implementation to be used